modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -122,6 +122,45 @@ It provides easy interface to:
         }
     @endcode
 
+
+    @defgroup highgui_winrt WinRT support
+
+    This figure explains new functionality implemented with WinRT GUI. The new GUI provides an Image control,
+    and a slider panel. Slider panel holds trackbars attached to it.
+
+    Sliders are attached below the image control. Every new slider is added below the previous one.
+
+    See below the example used to generate the figure:
+    @code
+        void sample_app::MainPage::ShowWindow()
+        {
+            static cv::String windowName("sample");
+            cv::winrt_initContainer(this->cvContainer);
```

Comment:
Can we "hide" this step in "initialization" internals of `namedWindow`/`imshow`/etc?